### PR TITLE
Limit Order UI improvement

### PR DESCRIPTION
**Pull request by hungdoansy**
1. Update wording
2. Implement get token price with a custom chainId
3. Show warning if insufficient fund
image

4. Get user to confirm if user's price is worse than market
image

5. Show note if user's price is better than market
image

### CODE REVIEW

File: src/components/swapv2/LimitOrder/LimitOrderForm.tsx [hunk 1]
```diff
@@ -615,11 +616,12 @@ const LimitOrderForm = function LimitOrderForm({
   const warningMessage = useMemo(() => {
     const messages = []
 
-    if (currencyIn && displayRate && !deltaRate.profit && deltaRate.percent) {
+    if (currencyIn && displayRate && !deltaRate.profit && Number(deltaRate.rawPercent) <= -5) {
```
Comment (nguyenhoaidanh) [inline src/components/swapv2/LimitOrder/LimitOrderForm.tsx:619]: -5 from spec?

File: src/components/swapv2/LimitOrder/Modals/ConfirmOrderModal.tsx [hunk 2]
```diff
@@ -45,8 +46,11 @@ export default memo(function ConfirmOrderModal({
   rateInfo: RateInfo
   note?: string
   warningMessage: ReactNode[]
+  percentDiff: number
 }) {
   const { account } = useActiveWeb3React()
+  const [confirmed, setConfirmed] = useState(false)
+  const shouldShowConfirmFlow = percentDiff < -5
```
Comment (nguyenhoaidanh) [inline src/components/swapv2/LimitOrder/Modals/ConfirmOrderModal.tsx:53]: move -5 to constant

File: src/components/swapv2/LimitOrder/LimitOrderForm.tsx (hunk 1 above)
Comment (hungdoansy) [reply src/components/swapv2/LimitOrder/LimitOrderForm.tsx]: Nice catch. Done

Comment (hungdoansy) [reply src/components/swapv2/LimitOrder/LimitOrderForm.tsx]: Ah yes, it's from specs

**Review by nguyenhoaidanh — DISMISSED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**